automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -55,6 +55,28 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   }
 
   def launchAnalysis(rootEntityType: String, entityId: String, expression: String = "", enableCallCaching: Boolean = true): SubmissionDetailsPage = {
+    // TODO remove after fix. short term workaround that unblock test execution. Jira Ticket is https://broadinstitute.atlassian.net/browse/GAWB-3711
+    Try(await condition {
+      logger.info("GAWB-3711 workaround: looking for enabled Launch Analysis button")
+      openLaunchAnalysisModalButton.isStateEnabled match {
+        case true =>
+          logger.info("GAWB-3711 workaround: End of try")
+          true
+        case false =>
+          // if message modal is present, close it then reloads page
+          val msgModal = new DisabledMessageModal()
+          if (msgModal.isVisible && msgModal.getMessageText.startsWith(msgModal.message)) {
+            msgModal.clickXButton()
+            go to webDriver.getCurrentUrl
+            Thread.sleep(1000)
+          }
+          logger.info("GAWB-3711 workaround: Retrying")
+          false
+      }
+    }).recover {
+      case _: Exception => throw new TimeoutException("Timed out looking for enabled Launch Analysis button (GAWB-3711 workaround).")
+    }
+
```

Comment:
I'm confused about the intention here. Do you plan to remove this workaround once the root cause is fixed? If so, I'd suggest tackling the root cause instead of putting this in at all. 

Or, if the intention is to use this workaround to gather data on what's happening with the root cause, that's fine with me, as long as you remember to remove it soon.

Reply:
GAWB-3711 issue is known but currently is in dev backlog and happens very rarely. I believe test can continue to run after page reload. My intentions is to test out my hypothesis.

Reply:
If workaround works, it will be helpful to add this new info in the Jira ticket.

Reply:
So, I'm fine with this workaround if it's just to gather data - I *don't* like this workaround as any kind of permanent solution, as it seems to just plaster over the root cause. So, all good if this is temporary.